apps/backend/src/modules/contributors/contributors.service.ts

```diff
@@ -0,0 +1,78 @@
+import { ExceptionMessage } from "~/libs/enums/enums.js";
+import { HTTPCode } from "~/libs/modules/http/http.js";
+import { type Service } from "~/libs/types/types.js";
+
+import { ContributorEntity } from "./contributor.entity.js";
+import { type ContributorRepository } from "./contributor.repository.js";
+import { ContributorError } from "./libs/exceptions/exceptions.js";
+import {
+	type ContributorCreateRequestDto,
+	type ContributorGetAllItemResponseDto,
+} from "./libs/types/types.js";
+
+class ContributorService implements Service {
+	private contributorRepository: ContributorRepository;
+
+	public constructor(contributorRepository: ContributorRepository) {
+		this.contributorRepository = contributorRepository;
+	}
+
+	public async create(
+		payload: ContributorCreateRequestDto,
+	): Promise<ContributorGetAllItemResponseDto> {
+		const { name } = payload;
```

Comment:
can we create git_emails through relation inside contributors service?

Reply:
I chose to stick with the old method of using gitEmailService.create. Just because updating the model or types in the contributors service create to handle sending correct format data(gitEmail with the nested contributor?) or finding again already created gitEmail in the activity log service would have made things pretty messier. It seemed cleaner for me to keep things as they are for now.

Reply:
good, I'm ok with this approach